etc/kayobe/inspector.yml

```diff
@@ -100,6 +100,16 @@ inspector_lldp_switch_port_interface_map:
 
 # List of additional ironic inspector rules.
 #inspector_rules_extra:
+inspector_rules_extra:
+  - description: "Set the expected BIOS version."
+    conditions:
+    - field: "data://inventory.system_vendor.product_name"
+      op: "matches"
+      value: "^PowerEdge R630$"
```

Comment:
Can we make this the same in alt-1 and production? Shouldn't have any negative consequences and reduces the diff and potential merge issues.

As it stands, all nodes should have the same BIOS version shouldn't they?

Reply:
Sounds good - I think it's the same for the R6 series, but doesn't that C series node have a different BIOS? Shouldn't be a problem either way.

Reply:
C4130 was a different binary but was also 2.6.0

Reply:
Thanks, I gave it it's own rule just in case it diverges in the future.